packages/bottom-tabs/src/views/BottomTabItem.tsx

```diff
@@ -120,28 +120,29 @@ type Props = {
 };
 
 export default function BottomTabBarItem({
-  focused,
   route,
+  href,
+  focused,
   label,
   icon,
   badge,
   badgeStyle,
-  to,
   button = ({
+    href,
     children,
     style,
     onPress,
-    to,
     accessibilityRole,
     ...rest
   }: BottomTabBarButtonProps) => {
-    if (Platform.OS === 'web' && to) {
+    if (Platform.OS === 'web') {
       // React Native Web doesn't forward `onClick` if we use `TouchableWithoutFeedback`.
       // We need to use `onClick` to be able to prevent default browser handling of links.
       return (
         <Link
           {...rest}
-          to={to}
+          href={href}
+          action={CommonActions.navigate(route.name, route.params)}
           style={[styles.button, style]}
           onPress={(e: any) => {
             if (
```

Comment:
Not to this PR but a general question - Why this and `LinkPressable` component in the DrawerItem isn't reused and extracted to the `elements` package but copied over instead?